### PR TITLE
Some additions

### PR DESCRIPTION
**Pull request by ghost**
Just some new stuff, especially for "things"

### CODE REVIEW

**Comment by ghost** [comment]
Okay according to contributing.md i might have done some very bad things which i should fix

File: prefixes.json [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (YouHaveTrouble) [inline prefixes.json]: random empty line?

File: suffixes.json [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (YouHaveTrouble) [inline suffixes.json]: random empty line?

**Comment by ghost** [comment]
Fixed those

**Review by YouHaveTrouble — APPROVED** [review]

**Comment by YouHaveTrouble** [comment]
you missed 2 commas in prefixes and suffixes, broke prod, already fixed that

**Pull request merged**